spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerObjectMapperImpl.java

```diff
@@ -76,7 +74,7 @@
 		builder.put(Timestamp.class, ValueBinder::toTimestampArray);
 		builder.put(ByteArray.class, ValueBinder::toBytesArray);
 
-		writeBuilderIterableMapping = builder.build();
+		iterablePropertyType2ToMethodMap = builder.build();
```

Comment:
Is that "2To" redundant ? 

Also, it looks like this commit broke the build !

Reply:
well, I'm mapping to a "to method"...it's awkward. Without the 2 it reads like iterablePopertyType to method map. I'll check on the build.